packages/react-interactions/events/src/dom/__tests__/FocusWithin-test.internal.js

```diff
@@ -336,6 +338,68 @@ describe.each(table)('FocusWithin responder', hasPointerEvents => {
         expect.objectContaining({isTargetAttached: false}),
       );
     });
+
+    it.experimental(
```

Comment:
These kinds of unit tests that involve the scheduler don't feel like they belong in the responders to me. Doesn't really matter right now since all this is going away eventually. But it makes me think about how we decide where the boundaries lie, as did your PR that rearranged the DOM event plugin system and its tests for the better

Reply:
Yeah, I thought that. It's best if this code moved out of Flare soon.